### PR TITLE
Add a file parser for parsing from the local file system.

### PR DESCRIPTION
**Pull request by jwir3**
When using Roboelectric (or other unit testing framework for Android that
does not set up a "real" Android environment), it's necessary to retrieve
files based on the local file system of the machine on which the tests are
run, rather than the virtual file system expected by the Android operating
system. This is due to the fact that, when running these types of tests,
most of the Android system is really just a set of mocks. As such, without
this file parser, the "assets", "res", and other system-level file system
paths are not set up and linked correctly, as the system calls that would
normally provide this information are mocked.

### CODE REVIEW

**Comment by jwir3** [comment]
@andrzejchm I now added a `UTF-8` encoding specifier when parsing local files to eliminate the findbugs warning.

**Comment by andrzejchm** [comment]
Tremendous bunch of work, I'll dig into that.

File: README.md [hunk 1]
```diff
@@ -145,6 +144,23 @@ RESTMockServer.enableLogging(RESTMockLogger)
 RESTMockServer.disableLogging()
 ```
 
+## Unit Tests with Robolectric
+If you want to write unit tests (no emulator or device necessary), you can use [http://robolectric.org](Robolectric)
```
Comment (andrzejchm) [inline README.md]: here it should be `(Robolectric)[http://robolectric.org]`

File: README.md [hunk 2]
```diff
@@ -11,7 +11,6 @@ RESTMockServer.whenGET(pathContains("users/defunkt"))
             .thenReturnFile(200, "users/defunkt.json");
 ```
 **Article**
-
 [ITDD - Instrumentation TDD for Android](https://medium.com/@andrzejchm/ittd-instrumentation-ttd-for-android-4894cbb82d37)
 ##Table of Contents
 - [About](#about)
```
Comment (andrzejchm) [inline README.md]: we should add a link to the section about Unit tests to ToC:
`- [Unit Tests with Robolectric](#unit-tests-with-robolectric)`
just before `Android sample project`

**Comment by jwir3** [comment]
@andrzejchm Sure, I'll make the requested documentation changes & re-submit the PR.

**Comment by jwir3** [comment]
@andrzejchm I made the changes to the README you requested.

**Comment by andrzejchm** [comment]
thanks! merged

**Pull request merged**